Rlog/View/Components/CustomTextFieldContainer.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CustomTextFieldContainer.swift
+//  MacCPlayGround
+//
+//  Created by Noah's Ark on 2022/10/19.
+//
+
+import SwiftUI
+
+struct CustomTextFieldContainer: View {
+    let containerType: CustomTextFieldType
+    let keyboardType: UIKeyboardType
+    var text: Binding<String>
+    
+    init(containerType: CustomTextFieldType, keyboardType: UIKeyboardType, text: Binding<String>) {
+        self.containerType = containerType
+        self.keyboardType = keyboardType
```

Comment:
```swift
    var keyboardType: UIKeyboardType {
        switch self {
        case .wage, .payday: return .numberPad
        case .workplace, .reason, .none: return .default
        }
    }
```

디자인을 보니까 type에 따라서, 키보드가 정해져있는듯해요 ! 
매번 init할 때마다 작성하는 것보다는 default 값을 넣어두면 어떨까요 ?

```suggestion
    init(containerType: CustomTextFieldType, keyboardType: UIKeyboardType? = nil, text: Binding<String>) {
        self.containerType = containerType
        self.keyboardType = keyboardType ?? containerType.keyboardType
```

Reply:
오우...! 🌝 조언해주신 내용으로 기본값을 설정하니 좀 더 코드를 생략할 수 있었습니다. 
[9e4c3d8](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/10/commits/9e4c3d8c67df1d4cadc5282b3d741ea2d26d881d) 으로 변경하였고, 텍스트 필드 선언 시 타입을 지정해주면 조건에 맞는 키보드가 활성화 될 수 있도록 세팅했습니다!

```swift
// 사용 예시
CustomTextField(textFieldType: .workplace, text: $text)
CustomTextFieldContainer(containerType: .workplace, text: $text2)
CustomTextFieldContainer(containerType: .payday, text: $text3)
```